source/_components/switch.recswitch.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Ankuoo Rec Switch"
+logo: ankuoo_recswitch.png
+description: "Instructions on how to integrate Ankuoo Rec Switch into Home Assistant."
+date: 2018-10-05 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: "0.81"
+ha_category: Switch
+ha_iot_class: "Local Polling"
+---
```

Comment:
Logo is missing.

Reply:
Created PR: #6629

Well spotted. Thanks.